tests/test_nwm_loading_utils.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+import filecmp
+
+from teehr.loading.nwm.utils import (
+    build_zarr_references,
+    check_dates_against_nwm_version,
+    build_remote_nwm_filelist,
+    generate_json_paths
+)
+from teehr.loading.nwm.const import (
+    NWM22_ANALYSIS_CONFIG,
+    NWM30_ANALYSIS_CONFIG,
+)
+
+TEST_DIR = Path("tests", "data", "nwm30")
+TEMP_DIR = Path("tests", "data", "temp")
+
+
+def test_point_zarr_reference_file():
```

Comment:
@samlamont I get a failure on this one.  Looks to be the same issue we had before where the files don't match exactly.  I don't know why this happens or if it matters...I guess not since the data ingest seems to work.  Not sure how to handle this.  Did you do something to address this when it happened before or is it in v0.3.0?

Reply:
hmm this succeeds for me locally.  This comparison method (`filecmp`) does appear in v0.3.0 as well (only in this function).  I can replace `filecmp` with another method to compare the two files that is less stringent.  I think `filecmp` will fail even if there is a single difference in whitespace between the two files.

Reply:
The files are actually different - more than just formatting.  I don't understand what all is in the JSON data so I can't judge whether it is meaningfully different or not, but they are different.  My guess is that we are maybe not running the exact same version of all the packages involved so we get small and not meaningful differences in the files.  Once we are using poetry to lock all versions maybe this will resolve.  Lets not spend time on it for now.